### DISCUSSION TITLE
HIP-991: Permissionless revenue-generating Topic Ids for Topic Operators

### DISCUSSION THREAD

**Discussion by steven-sheehy** [hiero-ledger/hiero-improvement-proposals#1054] [Ideas]
Discussion for https://hips.hedera.com/hip/hip-991

**Comment by steven-sheehy** [comment]
Prefer `ConsensusCustomFee` be named `TopicCustomFee` since the use of consensus is ambiguous without also including topic like `ConsensusCreateTopicTransaction` has.

**Reply by Neurone** [reply]
Makes sense to me. If @netopyr agrees, I can send a PR to update the name of `ConsensusCustomFee` and `ConsensusCustomFeeList` to `TopicCustomFee` and `TopicCustomFeeList`.
